### DISCUSSION TITLE
Link Your Assigned Issue Number in PR

### DISCUSSION THREAD

**Discussion by MonalikaPatnaik** [MonalikaPatnaik/UMatter#364] [Announcements]
I see, many contributors just made a PR without adding the issue link..It is hard for us to examine the related issue of your PR..I suggest to add your issue link in your PR description(#Issue Number)..otherwise it will not be merged!

**Comment by ghoshvidip26** [comment]
So after creating issue, can I work on the issue and raise PR by with respect to that issue?

**Reply by MonalikaPatnaik** [reply]
After creating your issue, just wait to be assigned it to you..once assigned under GSSOC'23..you can start working and submit a PR!
